apisix/plugins/grpc-transcode/proto.lua

```diff
@@ -77,20 +77,22 @@ local function compile_proto_bin(content)
         return nil
     end
 
+    local files = pb.decode("google.protobuf.FileDescriptorSet", content).file
```

Comment:
do we need to add new test cases for this?

Reply:
The test cases are https://github.com/apache/apisix/blob/beff2df3311d86260db75315af1974e8d7326217/t/plugin/grpc-transcode2.t#L236 and below